lib/xboxkrnl/xboxkrnl.h

```diff
@@ -4001,6 +4023,12 @@ XBAPI VOID DECLSPEC_NORETURN NTAPI HalReturnToFirmware
     IN FIRMWARE_REENTRY Routine
 );
 
+/**
+ * Registers or unregisters a shutdown notification which contains a function
+ * pointer that will be called on shutdown.
```

Comment:
nit: This is the only description comment which ends in a period.

Reply:
Actually I seem to have used periods a few times in the past, but the non-period variant is more common. Changed.